apps/web/google_docs.talon

```diff
@@ -0,0 +1,339 @@
+    key(cmd-ctrl-shift-k)
+(go | move) [to] table column next:
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-b)
+(go | move) [to] table column (last | previous):
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-v)
+(go | move) [to] (table row start | start [of] table row):
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-j)
+(go | move) [to] (table row end | end [of] table row):
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-l)
+(go | move) [to] table row next:
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-m)
+(go | move) [to] table row (last | previous):
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-g)
+table (exit | leave) | (exit | leave) table:
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-e)
+(go | move) [to] (next table | table next):
+    key(cmd-ctrl-shift-n)
+    key(cmd-ctrl-shift-t)
+(go | move) [to] ((previous | last) table | table (previous | last)):
+    key(cmd-ctrl-shift-p)
+    key(cmd-ctrl-shift-t)
+speak [table] column and row header:
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-h)
+speak [table] cell location:
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-n)
+speak [table] row header:
+    key(cmd-ctrl-shift-t)
+    key(cmd-ctrl-shift-r)
+(go | move) [to] (next heading | heading next):
+    key(cmd-ctrl-n)
+    key(cmd-ctrl-h)
+(go | move) [to] ((previous | last) heading | heading (previous | last)):
+    key(cmd-ctrl-p)
+    key(cmd-ctrl-h)
+(go | move) [to] (next heading one | heading one next):
+    key(cmd-ctrl-n)
+    key(cmd-ctrl-1)
+(go | move) [to] (next heading two | heading two next):
+    key(cmd-ctrl-n)
+    key(cmd-ctrl-2)
+(go | move) [to] (next heading three | heading three next):
+    key(cmd-ctrl-n)
+    key(cmd-ctrl-3)
+(go | move) [to] (next heading four | heading four next):
+    key(cmd-ctrl-n)
+    key(cmd-ctrl-4)
+(go | move) [to] (next heading five | heading five next):
+    key(cmd-ctrl-n)
+    key(cmd-ctrl-5)
+(go | move) [to] (next heading six | heading six next):
+    key(cmd-ctrl-n)
+    key(cmd-ctrl-6)
+(go | move) [to] ((previous | last) heading one | heading one (previous | last)):
+    key(cmd-ctrl-p)
+    key(cmd-ctrl-1)
+(go | move) [to] ((previous | last) heading two | heading two (previous | last)):
+    key(cmd-ctrl-p)
+    key(cmd-ctrl-2)
+(go | move) [to] ((previous | last) heading three | heading three (previous | last)):
+    key(cmd-ctrl-p)
+    key(cmd-ctrl-3)
+(go | move) [to] ((previous | last) heading four | heading four (previous | last)):
+    key(cmd-ctrl-p)
+    key(cmd-ctrl-4)
+(go | move) [to] ((previous | last) heading five | heading five (previous | last)):
+    key(cmd-ctrl-p)
+    key(cmd-ctrl-5)
+(go | move) [to] ((previous | last) heading six | heading six (previous | last)):
+    key(cmd-ctrl-p)
+    key(cmd-ctrl-6)
+(go | move) [to] (next media | media next):
+    key(cmd-ctrl-p)
+    key(cmd-ctrl-g)
+open explore tool: key(cmd-alt-shift-i)
+open dictionary: key(cmd-shift-y)
+start voice typing: key(cmd-shift-s)
```

Comment:
Should this also put talon to sleep?

Reply:
It should, to avoid conflicts. I've applied the change – Talon now goes to sleep when voice typing is activated.